build.ps1

```diff
@@ -32,50 +32,53 @@ Skips building binaries targeting Visual Studio "14" (released as Visual Studio
 .PARAMETER SkipVS15
 Skips building binaries targeting Visual Studio "15"
 
-.PARAMETER SkipSubModules
-Skips updating submodules
-
 .PARAMETER SkipTests
 Skips building and running unit-tests
 
 .PARAMETER SkipILMerge
 Skips creating an ILMerged nuget.exe
 
 .PARAMETER Fast
-Combination of SkipTests, SkipSubModules, and SkipILMerge
+Runs incremental build. Skips environment setup.
+
+.PARAMETER CI
+Indicates the build script is invoked from CI
 
 .EXAMPLE
 To run full clean build, e.g after switching branches:
 .\build.ps1 -CleanCache
 
-To run "incremental" fast build with no tests:
-.\build.ps1 -Fast
-
 To troubleshoot build issues:
 .\build.ps1 -Verbose -ErrorAction Stop
 #>
-[CmdletBinding(DefaultParameterSetName='RegularBuild')]
+[CmdletBinding()]
 param (
     [ValidateSet("debug", "release")]
-    [string]$Configuration = 'debug',
+    [Alias('c')]
+    [string]$Configuration,
     [ValidateSet("release","rtm", "rc", "rc1", "beta", "beta1", "beta2", "final", "xprivate", "zlocal")]
+    [Alias('l')]
     [string]$ReleaseLabel = 'zlocal',
+    [Alias('n')]
     [int]$BuildNumber,
+    [Alias('sr')]
     [switch]$SkipRestore,
+    [Alias('cc')]
     [switch]$CleanCache,
     [string]$MSPFXPath,
     [string]$NuGetPFXPath,
+    [Alias('sx')]
     [switch]$SkipXProj,
+    [Alias('s14')]
     [switch]$SkipVS14,
+    [Alias('s15')]
     [switch]$SkipVS15,
-    [Parameter(ParameterSetName='RegularBuild')]
-    [switch]$SkipSubModules,
-    [Parameter(ParameterSetName='RegularBuild')]
     [switch]$SkipTests,
```

Comment:
`$SkipTests` is unused now, right? Does it make sense to leave it and, by default, call `RunTests.ps1`? I am fine with either solution, but there should be no unused parameter.